### DISCUSSION TITLE
LTBEEF via console?

### DISCUSSION THREAD

**Discussion by SN-Reality** [3kh0/ext-remover#1372] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
is it possible for someone to make lt beef work in a new tab by using the console?

**Comment by ghost** [comment]
uhhhhhh...

**Comment by schoolexploitkid** [comment]
go to chrome://extensions, paste in exploit.js, and copy the ext ids

**Reply by SN-Reality** [reply]
extensions is blocked

**Reply by akabutnicer** [reply]
go to:
`chrome-extension://gndmhdcefbhlchkhipcnnbkcmicncehk/manifest.json`
then continue on

**Reply by SN-Reality** [reply]
what am i pasting into because the console doesn't work on that page

**Reply by akabutnicer** [reply]
yeah i forgot, true

**Reply by ghost** [reply]
Enable the flag "#extensions-on-chrome-urls" if you can then you can to use Rigtools to run code on chrome urls  or use Rigtools to do the  Sh0vel exploit .

**Reply by akabutnicer** [reply, marked as answer]
yeah nah, that's not how it works. and I know this cause I've tried sh0vel with rigtools. say you enabled the flag, then did the sh0vel instructions via an extension with viable permissions, would it make sense to execute to a script specifically allowed for **CHROME URLS** on an **EXTENSION URL**? in other terms, if you enabled a flag to allow code execution on chrome URLs, why would you use that privilege on an **EXTENSION URL** (which actually isn't even possible, from an extension standpoint that is). the correct method now that I'm thinking about it would be to use rigtools and launch `gndmhdcefbhlchkhipcnnbkcmicncehk` as an extension page (via pressing Q)

**Reply by ghost** [reply]
Oh ok I apologize I thought I knew something .

**Comment by SlitheringLizard1** [comment]
there is a way check out my discussion on it
